src/V3Region.cpp

```diff
@@ -0,0 +1,68 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+
+#include "config_build.h"
+#include "verilatedos.h"
+
+#include "V3Global.h"
+#include "V3Region.h"
+#include "V3Ast.h"
+
+#include <map>
+
+class RegionVisitor : public AstNVisitor {
```

Comment:
Every visitor added slows down Verilator.  I suspect these markings can eventually be done as part of some other existing visitor.

It's ok while experimenting to leave this as a separate visitor as generally makes it easier to edit/experiment.

It might even be we don't need to mark the nodes at all as it might turn out that whereever we eventually use these markings could just determine the region on-the-fly.  But we'll see how it ends up.